src/state.tsx

```diff
@@ -203,6 +203,9 @@ class GlobalState implements State {
     },
     attributes: [MonsterAttribute.DEMON],
     weakness: null,
+    immunities: {
+      burn: null,
+    },
```

Comment:
make sure to add a state migration in `updateImportedData` since the type doesn't allow `undefined`.